docs/src/hal/twopass.adoc

```diff
@@ -12,16 +12,13 @@
 
 == TWOPASS
 
-LinuxCNC 2.5 supports TWOPASS processing of LinuxCNC configuration files
-that can help in the modularization and readability of LinuxCNC files.
-(LinuxCNC files are specified in an LinuxCNC INI file in the LinuxCNC stanza
-as [LinuxCNC]HALFILE=filename).
+This section describes an option to use names not numbers to distinguish multiple occurences of a single module in a HAL file.
```

Comment:
You can use names without using TWOPASS which doesn't help loading the module twice. So this wording could be a bit misleading.

Reply:
That is how I understood TWOPASS. I addressed everything except for this one now, I think. Could you please correct what I came up with? I doubt that it was better before - otherwise I likely would have understood it better :)

Reply:
Maybe something like this:

This section describes an option to have multiple load-commands for multiple instances of the same component at different positions in the file or among different files.
Internally, this requires to read the HAL file twice, hence the name TWOPASS.
Supported since LinuxCNC version 2.5, the TWOPASS processing of LinuxCNC configuration files helps with their modularization and readability.
To recall, LinuxCNC configuration files are specified in a LinuxCNC INI file as `[HAL]HALFILE=filename`.

Reply:
@smoe Have you noticed this?